apps/condo/domains/user/access/UserExternalIdentity.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generated by `createschema user.UserExternalIdentity`
+ */
+const access = require('@open-condo/keystone/access')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadUserExternalIdentities (args) {
+    const { authentication: { item: user } } = args
+
+    if (!access.userIsAuthenticated(args)) {
+        return false
+    }
+
+    if (user.type === RESIDENT) {
+        // own identities read
+        return { user: { id: user.id } }
+    } else if (access.userIsAdminOrIsSupport(args)){
+        return true
+    } else {
+        return false
+    }
+}
+
+async function canManageUserExternalIdentities (args) {
+    const { authentication: { item: user }, originalInput } = args
+
+    if (!access.userIsAuthenticated(args)) {
+        return false
+    }
+
+    const isSoftDelete = access.isSoftDelete(originalInput)
```

Comment:
0) !user -> false
0.1) isaAdmin -> true (?)
1) soft-delete for self
2) return false

Reply:
done